scripts/report_scripts/index.Rmd

```diff
@@ -22,6 +22,18 @@ library(reshape2)
 df_var <- read.table(var_timecourse_csv, sep = "\t", header=T)
 df_mut <- read.table(mut_timecourse_csv, sep = "\t", header=T)
 ```
+PiGx ww sars-cov-2 performs mutation analysis of SARS-CoV-2 and reports and quantifies the occurrence of 
+variants of concern (VOC) and signature mutations by which they are characterised. For each sample three reports are 
+generated:  
+* a qc-report reporting general statistics and amplicon coverage
+* a variant-report which including tables summarizing the mutation calling and the deconvolution results for the abundance of VOCs
+* a *Kraken2* report including a krona plot showing the analysis of the unaligned reads
```

Comment:
a taxonomic classification report including a krona plot showing the analysis of the unaligned reads